membership_delegated_partner_line/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,10 @@
+[ This file is optional, it should explain how to configure
+  the module before using it; it is aimed at advanced users. ]
+
+This module is incompatible with `membership_delegated_partner` and any module that depends
+on it. Prior to installing this module, if you use membership_delegated_partner:
```

Comment:
Can't they coexist? A possible approach could be to use the `delegated_member_id` in `account.move` to force the values in the lines so we can take advantage of both use cases.

Reply:
Unfortunately not in a sane way, I originally tried to do that. Due to the overwrite of write and create in the upstream module and how they are done. Besides it doesn't make a lot of functional sense to have both IMO, as the point of this module is to add different members.